ops/terraform/env/mgmt/base_config/values/mgmt.yaml

```diff
@@ -6,8 +6,7 @@ bfd:
-          service_account_access_password: <<CIPHER>>AgV4OfWqCU4uOkHHLvAi6RWVS0pfEoFCIpp9-XDvCMHEfkQAdAACAAtBcHBsaWNhdGlvbgAGY2lwaGVyABVhd3MtY3J5cHRvLXB1YmxpYy1rZXkAREE0UG42MWM3ZTJQb3QvT3dOend1V1NzQ2RaK1BtcnprVU8weW5NZmU3L0NXUTYxc3lzNmtqemdUT1NkazBYU3dRZz09AAEAB2F3cy1rbXMAS2Fybjphd3M6a21zOnVzLWVhc3QtMTo1NzczNzM4MzE3MTE6a2V5L21yay05ODhkZTIzNjVhMzU0ODgzYmY4YjI1M2Y4ODVlMDE1MwC4AQIBAHh4qGAs-obFTy45bxksscUbbjJB3aZizspADO-R3lWduwH6wuMgn7ZELaih7dxiv7CDAAAAfjB8BgkqhkiG9w0BBwagbzBtAgEAMGgGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQMXA_oYMiSSUGvsccaAgEQgDvQorqW7PzZ6B1LvwhTaS03GO2SuK0mKP0XjiCo9WYdQDUIbilRjvlUAypcABMd7g6LZDZia9ek03fxWQIAABAA-NhCFJuRnGUn1BXZuYPaw51yNaK7SUEIji_C0KnWEHnHUbfCNc9KcU7FydK98Dii_____wAAAAEAAAAAAAAAAAAAAAEAAAAMTMv_TSHZw-PFMaQfYMESDDE8A3G2IrvAKgQGEwBnMGUCMBPJmLOoE-f2DvveKw46_GIUZvpg8fZ1E5b_IZ2tp4iMVpL8enP9RHbLcdAC7XxHMQIxAKhNnqWRfNUQSvLfGKDgiLIp5SdStiTBFh93GGxfZDtjUgfQ8lvWXgXQQrte6LmUWA==<</CIPHER>>
-          service_account_access_key: <<CIPHER>>AgV4BX5zeoSS0EVlKSZdwNGFuaLxa8OooEVKjc7JMHo6iD8AdAACAAtBcHBsaWNhdGlvbgAGY2lwaGVyABVhd3MtY3J5cHRvLXB1YmxpYy1rZXkAREFpK0E1aStMa2kvNGlRVDdEa2tlNDJZNzhYRFZBNjl6S0hzZUwzRGdlU2ppdCtLOVVHSUhCNkJkQTVBVzMyUFRXdz09AAEAB2F3cy1rbXMAS2Fybjphd3M6a21zOnVzLWVhc3QtMTo1NzczNzM4MzE3MTE6a2V5L21yay05ODhkZTIzNjVhMzU0ODgzYmY4YjI1M2Y4ODVlMDE1MwC4AQIBAHh4qGAs-obFTy45bxksscUbbjJB3aZizspADO-R3lWduwH0vzEppEtLUBP1M7t9X0sOAAAAfjB8BgkqhkiG9w0BBwagbzBtAgEAMGgGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQMd5TpRO--f2ahtP9tAgEQgDuWmOx2LkGC8c6Ubqnw9uF8NdEgxiPMAYa_meDDIvp-eWQmytJrPhfpf7QcZ0RLNHUZXK9tVAJf-nahHgIAABAAdfeJ555-kbQi_v78MdlLFGWpY9FFR29EZQsqYNdp0itywAQ7YPce6gHq-MzbBHha_____wAAAAEAAAAAAAAAAAAAAAEAAAAuio51UKTbJeKHnbK5T7GJ3D31Y15eMEKSj8lnEyYoYgFDTVp2BJm7lsNJmM_mGppDwsjLyLoqdLKTQw15BcYAZzBlAjBrvywwbbpqPEweYhFzT8SfjFS0_NaGQwD1OZv-34aL9rPVI8oyFNbECCskjqfWTjICMQDxczvoAqkGSvf6CK82EsprxVOxWxk1gA0BC3uwJ2m2lp0VFcWwIAoWpkv4HNKmAeM=<</CIPHER>>
+          service_account_access_key: <<CIPHER>>AgV4ZI9xVcMv6Gn2qGXTdHMiPG04hB-qfPRFusinOkEvUDMAdAACAAtBcHBsaWNhdGlvbgAGY2lwaGVyABVhd3MtY3J5cHRvLXB1YmxpYy1rZXkAREF3cnZBRzZQSXp3WldpVVJJL1RRVnc0cUhyb1Q5K1lrRDhIbndzMVlUb05vS0VHdmZUcE1OWmtNN0NyUXdzOFBJdz09AAEAB2F3cy1rbXMAS2Fybjphd3M6a21zOnVzLWVhc3QtMTo1NzczNzM4MzE3MTE6a2V5L21yay05ODhkZTIzNjVhMzU0ODgzYmY4YjI1M2Y4ODVlMDE1MwC4AQIBAHh4qGAs-obFTy45bxksscUbbjJB3aZizspADO-R3lWduwGudO9hgvkyZpZnt7QMJATrAAAAfjB8BgkqhkiG9w0BBwagbzBtAgEAMGgGCSqGSIb3DQEHATAeBglghkgBZQMEAS4wEQQMwHFsS-qIBr87vwaRAgEQgDv2CoRVN6Q2vXomMdWV3-7mjF0vrK4aeM2-y_T6U-KmGSxcgqeENhW4Aoq0H4VBb79Tf2hQhz3w9TZ1BAIAABAAiDvLcieB0cdbGduWchBTeYgjZ131eRVU2TTrO1NOhxcQIKmYzn2QkW4t4_afpTxB_____wAAAAEAAAAAAAAAAAAAAAEAAAAuuamY2mg8UpQaQp2jIHsZkxBVl4k9qaPXDh-fl9kIY1L1GNG7BVkScuxbD3fkf9pX6qoVhPSzfwihOQJ4tPEAZzBlAjAtAvyqSs8QBHvxbMuIAlhpzuThInEsz4-5kKQF_4bnksUWJ61wvYS8Q59MbTNJGpYCMQD3cgR6NuK_bO0EvV_wZDEecSnBhjtBsylIjmI-mXdsG8XKD7Hk3Y-fKYFHvcOs86A=<</CIPHER>>
```

Comment:
I think we left this at wanting to delete this from the yaml and remove it from the state, e.g. `terraform state rm 'aws_ssm_parameter.common_sensitive["/bfd/mgmt/common/sensitive/service_accounts/sonar/service_account_access_key"]'`. I think we could unblock @timothy-ellis-ky on his CI work under BFD-3384 in this fashion.

```suggestion
```